src/warning.rs

```diff
@@ -112,35 +112,104 @@ be an update. Remove the update or remove fields that need to be copied.")
                     .unwrap();
                 }
 
-                Warning::UnusedType { location, name } => {
+                Warning::UnusedType {
+                    location, imported, ..
+                } => {
+                    let title = if *imported {
```

Comment:
Can we have only the first word capitalised in these warning messages? To match the others